joomla/joomla-cms
# Add Bootstrap 3 alert-link class

Partial Pull Request for Issue #10604
#### Summary of Changes

Adds support for the Bootstrap 3 `alert-link` class to make links stand out more and implement it in the Install from Web alert
#### Testing Instructions

Apply the patch and navigate to `administrator/index.php?option=com_installer`; note that the links stand out more now.

Before:
<img width="1170" alt="screen shot 2016-05-30 at 11 50 39 am" src="https://cloud.githubusercontent.com/assets/368545/15654843/0fba4332-265d-11e6-99d9-77033211033b.png">

After:
<img width="1170" alt="screen shot 2016-05-30 at 11 50 55 am" src="https://cloud.githubusercontent.com/assets/368545/15654846/14f0b80e-265d-11e6-87e2-82bb37d01320.png">
